src/gp_config.c

```diff
@@ -538,6 +538,17 @@ static int load_services(struct gp_config *cfg, struct gp_ini_context *ctx)
                     goto done;
                 }
             }
+
+            cfg->svcs[n]->min_lifetime = 0;
```

Comment:
Can you please ad a DEFAULT_MIN_LIFETIME macro for this?
We might actually set a default min_lifetime that is not 0, thinking about it.

Reply:
Okay.  Have you thought about whether you want the default to be non-zero?